src/view-utilities.js

```diff
@@ -335,7 +335,116 @@ var viewUtilities = function (cy, options) {
     cy.autounselectify(false);
     marqueeZoomEnabled = false;
   };
+  var geometric = require('geometric')
+
+  instance.enableLassoMode = function ({callback = undefined, lineColour = "#d67614", lineWidth = 3} = {}) {
+    
+    var isClicked = false;
+    var tempCanv = document.createElement('canvas');
+    tempCanv.id = 'temporary-canvas';
+    const container = cy.container();
+    container.appendChild(tempCanv);
+    
+
+    const width = container.offsetWidth;
+    const height = container.offsetHeight;
+
+    tempCanv.width = width;
+    tempCanv.height = height;
+    tempCanv.setAttribute("style",`z-index: 1000; position: absolute; top: 0; left: 0;`,);
+    
+    cy.panningEnabled(false);
+    cy.zoomingEnabled(false);
+    cy.autounselectify(true);
+    var points = [];
+
+    tempCanv.onclick = function(event) {
+      
+      if(isClicked == false)  {
+        isClicked = true;
+        context = tempCanv.getContext("2d");
+        context.strokeStyle = "#d67614";
+        context.lineJoin = "round";
+        context.lineWidth = 3;
```

Comment:
Let's not hardcode strokeStyle and lineWidth values, instead use them from options.